### PR TITLE
Java low level networking project

### PR DESCRIPTION
**Pull request by egimaben**

### CODE REVIEW

File: core-java/src/main/java/com/baeldung/java/nio/selector/EchoClient.java [hunk 1]
```diff
@@ -6,9 +6,15 @@
 import java.nio.channels.SocketChannel;
 
 public class EchoClient {
+<<<<<<< HEAD
```
Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/java/nio/selector/EchoClient.java]: Merge conflict artifacts in this file in general

File: core-java/src/main/java/com/baeldung/java/nio/selector/EchoServer.java [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+import java.nio.channels.ServerSocketChannel;
+import java.nio.channels.SocketChannel;
+import java.nio.channels.Selector;
+import java.nio.channels.SelectionKey;
+import java.nio.ByteBuffer;
+import java.io.IOException;
+import java.util.Set;
+import java.util.Iterator;
+import java.net.InetSocketAddress;
+=======
```
Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/java/nio/selector/EchoServer.java]: and here :)

File: core-java/src/test/java/com/baeldung/java/nio/selector/EchoTest.java [hunk 3]
```diff
@@ -2,20 +2,32 @@
 
 import static org.junit.Assert.assertEquals;
 
+<<<<<<< HEAD
```
Comment (pivovarit) [inline core-java/src/test/java/com/baeldung/java/nio/selector/EchoTest.java]: and here :)

**Comment by egimaben** [comment]
hey @pivovarit  changes have been made, let me know if it works

**Pull request merged**